ui/containers/WorkspaceDetails/constants.ts

```diff
@@ -7,4 +7,5 @@ export const distinguishedNameRegEx = /(?=.*?(CN=))(?=.*?(OU=))(?=.*?(DC=))/i;
 
 export enum ProtocolTypes {
   HDFS = 'hdfs',
+  S3A = 's3a',
```

Comment:
protocol types shouldn't be an enum because we want to support unknown types. Just have hdfs or anything else

Reply:
please squash commit

Reply:
done